praxis-isncsci/ui
# Make color stories more readable

Closes #35

## Problem

In the color stories, it is hard to tell where one color starts and ends. It is also hard to tell the swatch range as cards are distributed through multiple rows.

## Solution

1. Add more spacing between color blocks
2. Add a component that shows all color swatches as a gradient

## Testing

- [x] 1. Color blocks are better defined

<details>
  <summary>Test case</summary>

  1. Open the color story `?path=/docs/designsystem-color--docs`
  2. Confirm that the color blocks are better defined

| Before | After |
|--------|------|
| ![Before](https://github.com/praxis-isncsci/ui/assets/1294355/2a6d899d-d1af-4bfd-a5f8-754b41e47773) | ![image](https://github.com/praxis-isncsci/ui/assets/1294355/7ad4ec99-c66d-4f0e-b248-b5a7293ba723) |

</details>
